nautobot/docs/models/dcim/powerfeed.md

```diff
@@ -16,6 +16,3 @@ Each power feed also defines the electrical characteristics of the circuit which
 * Voltage
 * Amperage
 * Maximum utilization (percentage)
-
-!!! info
-    The power utilization of a rack is calculated when one or more power feeds are assigned to the rack and connected to devices that draw power.
```

Comment:
It became the opening line in the new Rack Power Utilization in `power.md`. Since powerfeed.md was only imported in that one place it seemed redundant. I, essentially, upgraded it from a note to its own section on the main page.

Reply:
Ah, yes I was unaware of that integration. Maybe it should be a note on a Rack instead?